sdk-workflows/src/main/java/io/dapr/workflows/runtime/WorkflowRuntimeBuilder.java

```diff
@@ -55,4 +55,33 @@ public <T extends Workflow> WorkflowRuntimeBuilder registerWorkflow(Class<T> cla
 
     return this;
   }
+
+  /**
+   * Registers an Activity object.
+   *
+   * @param clazz the class being registered
+   * @param <T>   any WorkflowActivity type
+   */
+  public <T extends WorkflowActivity> void registerActivity(Class<T> clazz) {
+    this.builder = this.builder.addActivity(
+        new ActivityWrapper<>(clazz)
+    );
+  }
+
+
+  /**
+   * Returns a WorkflowRuntime object.
+   *
+   * @return WorkflowRuntime
+   */
+  public WorkflowRuntime getInstance() {
+    if (instance == null) {
+      synchronized (WorkflowRuntime.class) {
+        if (instance == null) {
+          instance = new WorkflowRuntime(this.builder.build());
+        }
+      }
+    }
+    return instance;
```

Comment:
This getInstance() method is the same as build() method, I suggest to call build() directly:

```java
 public WorkflowRuntime getInstance() {
    return build();
  }
```

Reply:
Another question is that why we need this getInstance() on WorkflowRuntimeBuilder, why not use build() method since this is a builder class ? 

And I checked the code and only found one reference in WorkflowRuntimeBuilderTest.

Reply:
Please don't forget for this comment.

Reply:
`getInstance()` was originally in the implementation. The previous PR has changed the implementation to use `build()`. It is now redundant to use `getInstance()`. I have removed it for clarity.